app/database/operator/app_data_operator/index.test.ts

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import DatabaseManager from '@database/manager';
+import {
+    isRecordAppEqualToRaw,
+    isRecordGlobalEqualToRaw,
+    isRecordServerEqualToRaw,
+} from '@database/operator/app_data_operator/comparator';
+import {
+    transformAppRecord,
+    transformGlobalRecord,
+    transformServersRecord,
+} from '@database/operator/app_data_operator/transformers';
+import {RawGlobal, RawServers} from '@typings/database/database';
+
+describe('', () => {
```

Comment:
Empty description

Reply:
lol this is your code hahahahahahahahaha..

can you please batch your comments ?